TextToSpeech/TextToSpeechJsonRpc.cpp

```diff
@@ -90,27 +90,27 @@ namespace Plugin {
 
     uint32_t TextToSpeech::SetACL(const JsonObject& parameters, JsonObject& response)
     {
-        CHECK_TTS_PARAMETER_RETURN_ON_FAIL("accesslist");
-        TTSLOG_INFO("SetACL request:%s\n",parameters["accesslist"].String().c_str());
-        JsonArray list = parameters["accesslist"].Array();
-        JsonArray::Iterator it = list.Elements();
-        while(it.Next())
-        {
-            JsonObject accesslist = it.Current().Object();
-            if (accesslist.HasLabel("method") && accesslist.HasLabel("apps"))
-            {
-                m_AccessMutex.lock();
-                AddToAccessList(accesslist["method"].String(),accesslist["apps"].String());
```

Comment:
I believe AddToAccessList() & HasAccess() are not obsolete and can be removed

Reply:
yes